src/core/datagram.h

```diff
@@ -38,6 +38,11 @@ typedef struct QUIC_DATAGRAM {
     //
     BOOLEAN SendEnabled : 1;
 
+    //
+    // Indicates that the peer accepts RELIABLE_RESET kind of frames, in addition to RESET_STREAM frames.
+    //
+    BOOLEAN PeerReliableResetEnabled: 1;
```

Comment:
This shouldn't go in the Datagram, but the connection.

Reply:
Updated! Will push later.